src/oatpp/core/data/mapping/type/Primitive.hpp

```diff
@@ -110,6 +110,10 @@ class String : public type::ObjectWrapper<base::StrBuffer, __class::String> {
     return *this;
   }
 
+  inline String& operator = (const std::string& str) {
+    return operator=(str.c_str()); 
```

Comment:
This won't work for binary data.

You have to use the following instead:

```cpp
inline String& operator = (const std::string& str) {
  m_ptr = base::StrBuffer::createShared(str.data(), str.size(), true)
  return *this;
}
```

Reply:
Hey @lganzzzo 
I assume these tests should be placed in "oatpp/test/oatpp/core/data/mapping/type/StringTest.cpp" right?
Also is there a way to easily run a single test? I can outcomment the others, but I would think you have a smarter way :slightly_smiling_face: ?

Reply:
Hey @sholmp,

Yes,- just comment out the other tests:)
I don't have a smarter solution for that.